server/repository/src/migrations/v2_01_00/activity_log.rs

```diff
@@ -0,0 +1,16 @@
+use crate::migrations::DATETIME;
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    if cfg!(feature = "postgres") {
+        sql!(
+            connection,
+            r#"
+                ALTER TYPE activity_log_type ADD VALUE 'ASSET_PROPERTY_CREATED';
```

Comment:
Is this right? Or just missing from the asset properties PR?

Reply:
Missing from the other PR I think. Should have commented